Example/CustomViewController.swift

```diff
@@ -16,7 +16,8 @@ class CustomViewController: UIViewController {
     
     var simulateLocation = false
 
-    var userIndexedRoute: IndexedRoute?
+    var userRouteResponse: RouteResponse?
```

Comment:
Maybe `var indexedUserRouteResponse: IndexedRouteResponse?` can be used?

Reply:
Good catch. Fixed